src/utils/mappingUtil.ts

```diff
@@ -145,7 +149,7 @@ export class MappingUtil {
         break;
       default:
         payload.createdById = payload.testerStaffId;
-        payload.createdByName = payload.testerName;
+        payload.createdByName = payload.testerName ?? '';
```

Comment:
are we confident this is equivalent? This used to set createdByName as null, now will set it as ''

Reply:
So with this one, previously payload values were...

```
  createdByName?: string;
  testerName: string;
```

So testerName would have always been populated, whereas now it can be string or undefined.